lua/entities/gmod_wire_gpu/cl_gpuvm.lua

```diff
@@ -954,339 +954,339 @@ end
+						x = (v1.x+v2.x) * 1/3,
+						y = (v1.y+v2.y) * 1/3,
+						z = (v1.z+v2.z) * 1/3
+					}
+
+					-- Compute normal (world space)
+					local x1 = v2.x - v1.x
+					local y1 = v2.y - v1.y
+					local z1 = v2.z - v1.z
+
+					local x2 = v3.x - v1.x
+					local y2 = v3.y - v1.y
+					local z2 = v3.z - v1.z
+
+					local normal = {
+						x = y1*z2-y2*z1,
+						y = z1*x2-z2*x1,
+						z = x1*y2-x2*y1
+					}
+
+					-- Normalize it
+					local d = (normal.x^2 + normal.y^2 + normal.z^2)^(1/2)+1e-7
+					normal.x = normal.x / d
+					normal.y = normal.y / d
+					normal.z = normal.z / d
+
+					-- Perform culling
+					if self.VertexCulling == 1 then
+						if self.Memory[65469] == 0 then
+							cullVertex = (normal.x*v1.x + normal.y*v1.y + normal.z*v1.z) <= 0
+						else
+							cullVertex = (normal.x*v1.x + normal.y*v1.y + normal.z*v1.z) >= 0
+						end
+					end
+
+					-- Perform vertex lighting
+					if (self.VertexLighting == 1) and (not cullVertex) then
+						-- Extra color generated by lights
+						local lightColor = { x = 0, y = 0, z = 0, w = 255}
+
+						-- Apply all lights (world space calculations)
+						for i=0,7 do
+							if self.Lights[i] then
+								local lightPosition = {
+									x = self.Lights[i].Position.x,
+									y = self.Lights[i].Position.y,
+									z = self.Lights[i].Position.z
+								}
+								local lightLength = (lightPosition.x^2+lightPosition.y^2+lightPosition.z^2)^(1/2)+1e-7
+								lightPosition.x = lightPosition.x / lightLength
+								lightPosition.y = lightPosition.y / lightLength
+								lightPosition.z = lightPosition.z / lightLength
+
+								local lightDot
+								if self.Memory[65468] == 0 then
+									lightDot = math.abs(lightPosition.x*normal.x +
+																			lightPosition.y*normal.y +
+																			lightPosition.z*normal.z)*self.Lights[i].Color.w
+								else
+									lightDot = math.max(0,self.Memory[65468]*(lightPosition.x*normal.x +
+																			lightPosition.y*normal.y +
+																			lightPosition.z*normal.z))*self.Lights[i].Color.w
+
+								end
+
+								lightColor.x = math.min(lightColor.x + self.Lights[i].Color.x * lightDot,255)
+								lightColor.y = math.min(lightColor.y + self.Lights[i].Color.y * lightDot,255)
+								lightColor.z = math.min(lightColor.z + self.Lights[i].Color.z * lightDot,255)
+							end
+						end
+
+						-- Modulate object color with light color
+						resultColor.x = (1/255) * resultColor.x * lightColor.x
+						resultColor.y = (1/255) * resultColor.y * lightColor.y
+						resultColor.z = (1/255) * resultColor.z * lightColor.z
+					end
+
+					-- Perform distance culling
+					if (self.DistanceCulling == 1) and (not cullVertex) then
+						local Infront = {}
+						local Behind = {}
+
+						local frontCullDistance = self.Memory[65470]
+						local K = -frontCullDistance
+
+						-- Generate list of vertices which go behind the camera
+						if v1.z - K >= 0
+						then Behind [#Behind  + 1] = v1
```

Comment:
Weird formatting, but could still be indended better

Reply:
In the source file just like that. Such formatting is used here many times.